vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -229,13 +233,31 @@ void ScrollViewShadowNode::updateProperties(winrt::Microsoft::ReactNative::JSVal
       if (valid) {
         ScrollViewUWPImplementation(scrollViewer).PagingEnabled(pagingEnabled);
       }
+    } else if (propertyName == "inverted") {
+      const auto [valid, inverted] = getPropertyAndValidity(propertyValue, false);
+      if (valid) {
+        m_isInverted = inverted;
+        m_viewChanger.Inverted(inverted);
+        ScrollViewUWPImplementation(scrollViewer).SetInverted(inverted);
+        if (inverted) {
+          scrollViewer.HorizontalAnchorRatio(1.0);
```

Comment:
Is this a possible implementation for `maintainVisibleContentPosition`? I remember you were looking at allowing a more complex API thoguh.

Reply:
This could more or less be used as a basis for maintainVisibleContentPosition, yes. Especially if we also include `overflowAnchor` props to fix virtualization.